client/my-sites/theme/main.jsx

```diff
@@ -1025,22 +1005,28 @@ class ThemeSheet extends Component {
 		);
 	};
 
-	renderPreviewButton = () => {
+	renderPreviewButton = ( context ) => {
 		const { translate, isWpcomTheme, isExternallyManagedTheme } = this.props;
 		const isExternalLink = ! isWpcomTheme || isExternallyManagedTheme;
 
+		let classNames = 'theme__sheet-demo-button';
+		switch ( context ) {
+			case 'web-preview':
+			case 'screenshot':
+				classNames = 'theme__sheet-preview-demo-site';
+				break;
+		}
+
 		if ( ! this.shouldRenderPreviewButton() ) {
 			return null;
 		}
 
 		return (
 			<Button
-				className="theme__sheet-demo-button"
-				onClick={ ( e ) => this.previewAction( e, 'link', 'preview', 'regular' ) }
```

Comment:
Is it intended that we drop the third argument `regular`?

Reply:
`previewAction()` only accepts 3 arguments, I'm not sure why there's a fourth there...

https://github.com/Automattic/wp-calypso/blob/2cf726151df5524eea8952a51da35e06bd6271c3/client/my-sites/theme/main.jsx#L364

Reply:
OMG, I misread it 🤦